src/scripts/cloneRepo.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Script to clone a github repository
+
+username=$1
+repoName=$2
+
+echo $1 $2
+
+temp_sock=$(cat ./tmpAgent/agentInfo)
+export SSH_AUTH_SOCK=$temp_sock
+
+mkdir ./myRepos
+cd myRepos
```

Comment:
This will need further error handling in case the myRepos folder already exists.